modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/com/liferay/frontend/data/set/views/web/batch/data-set-views.json

```diff
@@ -297,9 +297,9 @@
 				{
 					"deletionType": "prevent",
 					"label": {
-						"en_US": "Entry Views"
+						"en_US": "FDS Entry Views"
 					},
-					"name": "entryViews",
+					"name": "fdsEntryViews",
```

Comment:
@seltong the prefix should be in uppercase letter. `FDSEntryViews`

Apply also for the others

Reply:
I was using [this](https://github.com/liferay/liferay-portal/blob/b0f9ff167391a91351f64198cbc567acdbf1f29f/modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-2/src/main/resources/site-initializer/object-relationships/test-object-relationship-2.json#L6) example as a base